styles/theme.scss

```diff
@@ -18,6 +18,7 @@ SPDX-License-Identifier: Apache-2.0 */
   --opacity-disabled: 0.3;
   --transparent-color: rgb(0 0 0 / 0%);
   --transition-duration: 0.15s;
+  --interface-maximum-width: 1550px;
```

Comment:
I don't think this should be editable in the theme. It is a magic number that is a part of the design of staking dashboard. I think it would be better as an SCSS variable (it is probably used in media queries also, in which case it will have to be an SCSS variable).